community/contribute/adding-docs-guide.md

```diff
@@ -23,34 +23,37 @@ Your repository containing the documentation has to...
 
 The documentation files have to be in markdown format and...
 
-- comply with [SCS licensing guidelines](../github/dco-and-licenses.md)
+- comply [SCS licensing guidelines](https://github.com/SovereignCloudStack/docs/blob/main/community/github/dco-and-licenses.md)
 - match our
-  - [markdown file structure guideline](../contribute/doc-files-structure-guide-md)
+  - [markdown file structure guideline](https://github.com/SovereignCloudStack/docs/blob/main/community/contribute/doc-files-structure-guide-md)
   - linting Rules
-  - [styleguide](../contribute/styleguide.md)
+  - [styleguide](https://github.com/SovereignCloudStack/docs/blob/main/community/contribute/styleguide.md)
 
 ### Step 2 – Adding your repo to the docs.json
 
 File a Pull Request within the [docs-page](https://github.com/SovereignCloudStack/docs-page) repository and add your repo to the docs.package.json:
 
 ```json5
 [
-    {
-        // link to github organisation and repository 
-        "repo": "demo-organisation/demo-repository",
-        // directory which shall be copied. Optional to specify only copying markdown files
-        "source": "doc/*.md",
-        // directory where the files should be copied to within the docs-page repo
-        "target": "docs",
-        // label for directory. only mandatory if source file is set to copy only *.md files and not the complete directory
-        "label": "demo-repository-label"
-    }
+  {
+    repo: "demo-organisation/demo-repository",
```

Comment:
I think having keys without `"` only works in JS (and JSON5, YAML...), not in JSON.
```suggestion
    "repo": "demo-organisation/demo-repository",
```